Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -797,15 +802,27 @@ extension SwiftLanguageService {
       additionalParameters: additionalCursorInfoParameters
     )
 
-    return cursorInfoResponse.refactorActions.compactMap {
+    var canInlineMacro = false
+
+    var refactorActions = cursorInfoResponse.refactorActions.compactMap {
       do {
         let lspCommand = try $0.asCommand()
+        canInlineMacro = $0.title == "Inline Macro"
```

Comment:
Instead of checking the macro title, what do you think about comparing the `actionString`. That should be more stable than the title.